bundles/org.openhab.binding.nzwateralerts/README.md

```diff
@@ -0,0 +1,69 @@
+# NZ Water Alerts Binding
+
+Get Water Alert Levels for cities in New Zealand. Getting this alert level can help you script and automate smarter tasks for water and avoid getting penalized from your distract or local council. 
```

Comment:
Please put each sentence on a single line. It will make seeing review diffs easier.